File: example/src/multiple-files/Class2.cpp
/*
Copyright (c) 2017 Fernando Bevilacqua <dovyski@gmail.com>
Licensed under the MIT license.
*/

#include "cvui.h"
#include "Class2.h"

Class2::Class2() {
}

void Class2::renderMessage(cv::Mat & frame) {
	cvui::text(frame, 10, 10, "Message");
}

File: example/src/multiple-files/Class2.py
#
# This is a dummy class to illustrate the use of cvui in a project
# that contains multiple files.
#
# Copyright (c) 2018 Fernando Bevilacqua <dovyski@gmail.com>
# Licensed under the MIT license.
#

import cvui

class Class2:
	def renderMessage(self, frame):
		cvui.text(frame, 10, 10, 'Message')
